src/locales/locale-en.json

```diff
@@ -18,7 +18,7 @@
       },
       "DEMOCRATIC": {
         "DESCRIPTION": "Karrot is made by its users: a free and open-source software developed with the feedback and participation of the people who use it. It is mainly developed by a small core team that is constantly in contact with people in different groups, in order to make sense of their diverging and converging needs and thus guide the development. We abide by the ethics of integrity and non-commercialization of data.",
-        "DESCRIPTION2": "Developers and designers are obviously welcome to participate in its development. Check out the project on {code}.",
+        "DESCRIPTION2": "Developers and designers are obviously welcome to participate in its development.",
```

Comment:
As this line no longer uses the `{code}` variable, it can be removed from the usage too.

So this:

https://github.com/yunity/karrot-frontend/blob/18cff7b940f66bddb3f80e0f5baf40e336b87dcc/src/base/pages/Landing.vue#L106-L114

and be changed to:

```html
<p v-t="'ABOUT_KARROT.SECTIONS.DEMOCRATIC.DESCRIPTION2'" />
```